ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -169,3 +169,209 @@ def verification_ui(self):
         for err in self.err_list:
             logger.error(err)
         assert len(self.err_list) == 0, f"{self.err_list}"
+
+    def odf_overview_ui(
+        self,
+    ):
+        """
+        Test to verify changes and validate elements on ODF Overview tab for ODF 4.9
+
+        """
+
+        self.ocp_version = get_ocp_version()
+        ocs_version = version.get_semantic_ocs_version_from_config()
+        if self.ocp_version >= "4.9" and ocs_version >= version.VERSION_4_9:
+
+            self.page_nav = locators[self.ocp_version]["page"]
+            self.dash_items = locators[self.ocp_version]["dashboard"]
+            self.dep_loc = locators[self.ocp_version]["deployment"]
+
+            self.navigate_installed_operators_page()
+            logger.info("Click on project dropdown")
+            self.do_click(self.dep_loc["project-dropdown"])
+            logger.info("Search for 'openshift-storage' project")
```

Comment:
```Show default projects``` option in project drop down will be disabled by default. So openshift-storage will not be present in the list